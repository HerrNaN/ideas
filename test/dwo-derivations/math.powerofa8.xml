<request service="derivation" exerciseid="math.powerofa" encoding="string">
  <state>
    <expr>
      1/a^2*(1/a^4)
    </expr>
  </state>
</request>
