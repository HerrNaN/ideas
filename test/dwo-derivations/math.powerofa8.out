<reply result="ok" version="0.5.11">
  <list>
    <elem ruleid="reciprocal">
      <expr>
        a^(-2)*(1/a^4)
      </expr>
      <context>
        <item name="location" value="[0]"/>
      </context>
    </elem>
    <elem ruleid="reciprocal">
      <expr>
        a^(-2)*a^(-4)
      </expr>
      <context>
        <item name="location" value="[1]"/>
      </context>
    </elem>
    <elem ruleid="add exponents">
      <expr>
        a^(-6)
      </expr>
    </elem>
  </list>
</reply>

